src/main/java/duke/exceptions/InvalidKeyException.java

```diff
@@ -0,0 +1,15 @@
+package duke.exceptions;
+
+/**
+ * InvalidKeyException is an exception that indicates that the key word is invalid.
+ */
+public class InvalidKeyException extends Exception{
```

Comment:
Minor detail but perhaps add a space before the curly brace? Same for other parts of the code.